medium/users/services.py

```diff
@@ -1,58 +1,59 @@
-from typing import Any, Dict
+from typing import Dict
 
 from django.contrib.auth.hashers import make_password
-from django.contrib.auth.models import BaseUserManager
 from django.contrib.auth.password_validation import validate_password
-from django.core.exceptions import ValidationError as DjValidationError
-from django.core.validators import validate_email
-from rest_framework.exceptions import ValidationError
 
+from medium.common.services import model_update
 from medium.users.models import User
 
+# from django.core.exceptions import PermissionDenied
+
 
 class UserService:
     def create_user(
-        self, username: str, name: str, email: str, password: str, **other_fields
```

Comment:
* is very dangerous, not readable, can be abused, make it hard to test , ..... 
 (*args, **kwargs)

you can use dataclasss as parameter handler